apps.yml

```diff
@@ -1017,10 +1017,7 @@ apps:
     - /iam-grafana
 - application:
     authorized_groups:
-    - mozilliansorg_nda
     - mozilliansorg_boxcom-access
-    - team_moco
-    - team_mofo
     - team_office_support
 #    - team_mzla
```

Comment:
Nitpick, let's strike the mzla commented line.  The purpose of this PR is to remove team_moco type groups.  The commented line indicates mzla might ever get added... which is now wrong.